tds-dll-common/src/main/java/tds/dll/common/rtspackage/student/data/Accommodation.java

```diff
@@ -66,6 +71,10 @@
   protected String          streamlinedInterface;
   @XmlElement (name = "IllustrationGlossary")
   protected String          illustrationGlossary;
+  @XmlElement (name = "EmbossRequestType")
+  protected String          embossRequestType;
+  @XmlElement (name = "BrailleType")
+  protected String          brailleType;
```

Comment:
I defer to you if it is worth changing but the properties below this one seem to be misaligned.  Or maybe it is just GitHub